internal/querynode/data_sync_service.go

```diff
@@ -36,257 +36,144 @@ const (
 	loadTypePartition  loadType = 1
 )
 
-// dataSyncService manages a lot of flow graphs for collections and partitions
+// dataSyncService manages a lot of flow graphs
 type dataSyncService struct {
 	ctx context.Context
 
-	mu                        sync.Mutex                                   // guards FlowGraphs
-	collectionFlowGraphs      map[UniqueID]map[Channel]*queryNodeFlowGraph // map[collectionID]flowGraphs
-	collectionDeltaFlowGraphs map[UniqueID]map[Channel]*queryNodeFlowGraph
-	partitionFlowGraphs       map[UniqueID]map[Channel]*queryNodeFlowGraph // map[partitionID]flowGraphs
+	mu              sync.Mutex                      // guards FlowGraphs
+	dmlFlowGraphs   map[Channel]*queryNodeFlowGraph // map[collectionID]flowGraphs
+	deltaFlowGraphs map[Channel]*queryNodeFlowGraph // map[collectionID]flowGraphs
 
 	streamingReplica  ReplicaInterface
 	historicalReplica ReplicaInterface
 	tSafeReplica      TSafeReplicaInterface
 	msFactory         msgstream.Factory
 }
 
-// collection flow graph
-// addCollectionFlowGraph add a collection flowGraph to collectionFlowGraphs
-func (dsService *dataSyncService) addCollectionFlowGraph(collectionID UniqueID, vChannels []string) {
+// addDMLFlowGraphs add a flowGraph to dmlFlowGraphs
+func (dsService *dataSyncService) addDMLFlowGraphs(collectionID UniqueID, partitionID UniqueID, loadType loadType, vChannels []string) {
```

Comment:
Is it better to channge the func name  addDMLFlowGraphs to addFlowGraphForDmChannel?

Reply:
fixed